index.html

```diff
@@ -384,6 +384,27 @@
         //     fx :        2.5,
         // });
 
+        itowns.viewer.addImageryLayer({
+                                        url       : "https://wxs.ign.fr/gratuit/geoportail/v/wms?",
```

Comment:
`WMS_Provider` uses `layer.url + "?SERVICE=WMS…"` which results in a doubled `??` in the URL. That doesn't seem to cause problems to the server but is probably not what was intended.

I don't know WMS enough but either the `?` here should be removed, or `WMS_Provider` should look at whether `layer.url` contains a `?` already and conditionally append with a `?` or a `&` (or possibly no separator if `layer.url` ends with one of those)